src/lib/leaflet-utils/map-factory.js

```diff
@@ -21,8 +22,10 @@ export default function (el, vnode, config) {
   // map.addLayer(baseLayers[INITIAL_BASELAYER])
   map.setZoom(config.zoom || mapConfig.zoom)
 
+  map.addControl(roseControl(map))
```

Comment:
Was L.rose.control not working properly?

Reply:
No it didn't work when imported, and also the provided images were nog satisfactory